examples/scaleway-machinedeployment.yaml

```diff
@@ -61,4 +61,4 @@ spec:
           operatingSystemSpec:
             disableAutoUpdate: true
       versions:
-        kubelet: 1.21.5
+        kubelet: 1.23.0
```

Comment:
Minor nit, let's make this `1.22.5` as with the other examples.

Reply:
Good catch :grin: